### PR TITLE
Chore/upgrade angular cli

### PR DESCRIPTION
**Pull request by chrisrhoden**

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -109,6 +109,15 @@
...
+    "invoke-injectable": true,
+    "callable-types": true,
+    "import-blacklist": [true, "rxjs"],
+    "import-spacing": true,
+    "interface-over-type-literal": true,
+    "no-empty-interface": true,
+    "no-string-throw": true,
+    "prefer-const": true,
+    "typeof-compare": true,
+    "unified-signatures": true,
```
Comment (cavis) [inline tslint.json]: Extra comma.  Tslint also complaining about some extra defs in here:

```
Could not find implementations for the following rules specified in the configuration:
    label-undefined
    no-duplicate-key
    no-unreachable
    directive-selector-prefix
    component-selector-prefix
    directive-selector-name
    component-selector-name
    directive-selector-type
    component-selector-type
Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.
```

Comment (cavis) [inline tslint.json]: Also ... not expecting you to fix all the lint errors in this PR.  Because `prefer-const` causes a ton of churn!

**Comment by cavis** [comment]
@chrisrhoden can you confirm/deny the existence of said `npm run build` bug?

**Comment by gcampo88** [comment]
@cavis I'm getting those errors too during `npm run build`, for whatever it's worth!

**Comment by chrisrhoden** [comment]
@cavis @gcampo88  my guess is that you need to yarn install. I just pulled the tslint config from the recommendations made by the angular-cli repo, we can definitely change them.

**Comment by chrisrhoden** [comment]
I fixed merge conflicts, all of the lint issues and ensured that the build was running. I am not sure if the tests were previously working correctly, if I need to dig further there I am happy to. @cavis let me know if there's more needed here.

**Comment by chrisrhoden** [comment]
https://gist.github.com/chrisrhoden/fa1cb95bc7f3cac3cd4c64108091843b

**Comment by cavis** [comment]
Huh - okay, so this works when I `npm install` as in your gist, but fails when I `yarn install`.  Guessing the yarn.lock is just out of date?  Also seems to be missing a karma reporter.  I may just merge this and clean it up in master.  (Need to clean up our dockerfile to do yarn-install as well).

**Comment by chrisrhoden** [comment]
@cavis appreciate the merge, but please let me know if there's a piece of this I can work on, I know y'all have a ton of priorities at PRX

**Pull request merged**